[2024-01-18T22:03:42.166+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_X_mention_score.train_model_theta scheduled__2024-01-01T01:30:00+00:00 [queued]>
[2024-01-18T22:03:42.182+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_X_mention_score.train_model_theta scheduled__2024-01-01T01:30:00+00:00 [queued]>
[2024-01-18T22:03:42.182+0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T22:03:42.183+0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2024-01-18T22:03:42.183+0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T22:03:42.208+0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): train_model_theta> on 2024-01-01 01:30:00+00:00
[2024-01-18T22:03:42.225+0300] {standard_task_runner.py:55} INFO - Started process 5668 to run task
[2024-01-18T22:03:42.237+0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'Google_X_mention_score', 'train_model_theta', 'scheduled__2024-01-01T01:30:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/google_mention_score.py', '--cfg-path', '/tmp/tmpwf9ki9sv']
[2024-01-18T22:03:42.244+0300] {standard_task_runner.py:83} INFO - Job 27: Subtask train_model_theta
[2024-01-18T22:03:42.344+0300] {task_command.py:389} INFO - Running <TaskInstance: Google_X_mention_score.train_model_theta scheduled__2024-01-01T01:30:00+00:00 [running]> on host xflow
[2024-01-18T22:03:42.440+0300] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=Google_X_mention_score
AIRFLOW_CTX_TASK_ID=train_model_theta
AIRFLOW_CTX_EXECUTION_DATE=2024-01-01T01:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-01T01:30:00+00:00
[2024-01-18T22:03:42.445+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-18T22:03:42.446+0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/angry/mlops_4/scripts/train_model.py theta']
[2024-01-18T22:03:42.465+0300] {subprocess.py:86} INFO - Output:
[2024-01-18T22:03:55.378+0300] {subprocess.py:93} INFO - Registered model 'theta_model' already exists. Creating a new version of this model...
[2024-01-18T22:03:55.423+0300] {subprocess.py:93} INFO - Created version '4' of model 'theta_model'.
[2024-01-18T22:03:56.123+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-18T22:03:56.164+0300] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=Google_X_mention_score, task_id=train_model_theta, execution_date=20240101T013000, start_date=20240118T190342, end_date=20240118T190356
[2024-01-18T22:03:56.217+0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-18T22:03:56.260+0300] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
